dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -803,8 +832,8 @@ def test_profile(self):
             },
             times=defaultdict(float, {'histogram_and_quantiles': 1.0,
                                       'precision': 1.0, 'max': 1.0, 'min': 1.0,
-                                      'skewness': 1.0, 'kurtosis': 1.0,
-                                      'sum': 1.0, 'variance': 1.0,
+                                      'skewness': 1.0,
```

Comment:
any reason for the move here?

Reply:
Must've been an accidental refactor, will revert